resources/views/lodge.blade.php

```diff
@@ -118,7 +118,11 @@
                     <div class="lodges">
                         @foreach ($accomodations as $accomodation)
                             <div class="lodge-card">
-                                <img src="{{ asset($accomodation->attachment[0]->url()) }}" alt="">
+                            @foreach ($accomodation->attachment as $image)
```

Comment:
You are running a loop for something that would use only the first item?

If you want introduce a check instead like an if and just access the first element. I see this causing problems.